staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -109,3 +122,13 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 		},
 	}
 }
+
+func stripEscapes(s string) string {
+	return strings.Map(func(r rune) rune {
+		switch {
+		case r == '"', r == '\\', int(r) < 32:
```

Comment:
comment on the source of this character class (JSON spec, I assume?)

Reply:
done. It's what `jsoniter.ConfigFastest` does not understand from the JSON ecma standard string format. Added a comment.